### ISSUE TITLE
Accessibility violation in datepicker button

### ISSUE THREAD

**Issue by markusv** [jquense/react-widgets#530]
Hi,

I am using the react-widgets datepicker in my project and am getting an accessibility violation for the button

<button tabindex="-1" title="" type="button" aria-disabled="false" aria-label="" class="rw-btn-calendar rw-btn"><i aria-hidden="true" class="rw-i rw-i-calendar"></i></button>

The error message is:

Buttons must have discernible text that clearly describes the destination, purpose, function, or action for screen reader users.

**Comment by jquense** [comment]
not sure which button this is referring to. They should all have labels and assosiative text already
